graph/formats/dot/fuzz/fuzz.go

```diff
@@ -0,0 +1,33 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build gofuzz
+
+package fuzz
+
+import (
+	"bytes"
+	"os/exec"
+
+	"gonum.org/v1/gonum/graph/formats/dot"
+)
+
+func Fuzz(data []byte) int {
```

Comment:
Perhaps not necessary, but add a doc comment just describing what `Fuzz` does with the given data, and what is returned.

Reply:
I'll just add a link to github.com/dvyukov/go-fuzz.